### ISSUE TITLE
`Use fewer imports` suggestion importing multiple fields from the same record

### ISSUE THREAD

**Issue by ChefYeum** [ndmitchell/hlint#1133]
For instance, for the following line of code:
`import Test.Hspec.Formatters (Formatter(failedFormatter), Formatter(footerFormatter))`

Suggest the following as alternative:
`import Test.Hspec.Formatters (Formatter(failedFormatter, footerFormatter))`

Does it sound like a reasonable feature to add under `Use fewer imports` suggestion?

**Comment by ndmitchell** [comment]
Sounds reasonable - PR most welcome!

**Comment by batkot** [comment]
This issue will soon get 3yo, but if it's still reasonable, I'll give it a shot :sweat_smile:

**Comment by zliu41** [comment]
@batkot Yes this is still relevant! HLint has a "Use fewer imports" warning when the same module is imported multiple times, but not in this case.

**Comment by Morabbin** [comment]
I have a better PR for this now, in draft: https://github.com/ndmitchell/hlint/pull/1654.

## To sort or not to sort?
Note that sorting occurs only when field imports are combined:
```haskell
import A (Foo(A,B), Bar(C,D), Foo(E), Foo(C,D))
import A (Foo(A,F), Baz(D,A))
```
yields the following hint:
```haskell
import A (Foo(A,B,C,D,E,F), Bar(C,D), Baz(D,A))
```
`Foo`'s field list is sorted. Note though that `Baz`'s is not, nor is the import list as a whole sorted wrt the type, class, or constructor names imported. The former is straightforward (don't change `Baz`'s import), but the latter requires a kind of stable nub.

The question is: do we prefer sorting any of these lists, or preserving the order of the original?

Sorting everything is simpler (`groupSortOn` FTW), but I have a feeling this is the hlint way?
